doc/release-notes.md

```diff
@@ -30,8 +30,12 @@
       
     - Added `consensus_wgs_plus_cnvkit_wxs.tsv.gz` which is a merge of `consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz` and `consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz` per [ticket 161](https://github.com/PediatricOpenTargets/ticket-tracker/issues/161)
     - Updated `ensg-hugo-rmtl-mapping.tsv` file per [ticket 146](https://github.com/PediatricOpenTargets/ticket-tracker/issues/146). The previous release of this file does not contain all gene ENSG IDs and symbols that are present in `snv-consensus-plus-hotspots.maf.tsv.gz`. This update merged GENCODE V28 and V38 to allow inclusion of more gene ENSG IDs and symbols. 
+    - Futher update `ensg-hugo-rmtl-mapping.tsv` [PR 48 D3b codes](https://github.com/d3b-center/D3b-codes/pull/48)
```

Comment:
```suggestion
    - Futher update `ensg-hugo-rmtl-mapping.tsv` [PR 48 D3b codes](https://github.com/d3b-center/D3b-codes/pull/48) to include all gene ENSG ID to symbol mappings in v7 `ensg-hugo-rmtl-mapping.tsv`.
```

Thank you for updating the release notes. Just adding a suggestion to clarify this update.

Reply:
@logstar, thanks so much! I will implement your suggested changes. One quick question - now that ticket 135 is in, do you want to add `all cohort` independent samples in the release as well? If so, which ones?

Reply:
Thank you for the question. Yes. It would be great to have `all-cohorts` independent sample lists in the release as well. All corresponding ones for `each-cohort` independent sample lists in slack message <https://opentargetspediatrics.slack.com/archives/C02B5K2B99C/p1629165953031500?thread_ts=1629158109.022400&cid=C02B5K2B99C> should be sufficient. If not, we could add more in v9.